src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3134,7 +3134,7 @@ static char* allocate_pages_individually(size_t bytes, char* addr, DWORD flags,
                                 PAGE_READWRITE);
   // If reservation failed, return null
   if (p_buf == nullptr) return nullptr;
-  MemTracker::record_virtual_memory_reserve((address)p_buf, size_of_reserve, CALLER_PC);
+  MemTracker::record_virtual_memory_reserve((address)p_buf, size_of_reserve, CALLER_PC, mtInternal);
```

Comment:
I think that allocate_pages_individually should take a MEMFLAGS argument instead of using mtInternal here.

Reply:
It takes now. All corresponding functions in call hierarchy are also changed.